pkg/cmd/grafana-cli/commands/reset_password_command.go

```diff
@@ -2,18 +2,26 @@ package commands
 
 import (
 	"fmt"
+	"syscall"
 
 	"github.com/fatih/color"
 	"github.com/grafana/grafana/pkg/bus"
 	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/util"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
 const AdminUserId = 1
 
 func resetPasswordCommand(c CommandLine) error {
-	newPassword := c.Args().First()
+	fmt.Print("New password: ")
+	bytePassword, err := terminal.ReadPassword(syscall.Stdin)
```

Comment:
Will this be backward compatible to `grafana-cli admin reset-admin-password <password>`?